### DISCUSSION TITLE
Volumen OSD, Perfíl de volumen, FTP, eliminar juegos

### DISCUSSION THREAD

**Discussion by Nico1942** [Rparadise-Team/Koriki#39] [Ideas & Requests]
Hola, ¿Cómo están?

Primero gracias por el sistema, está genial y me funciona bastante rápido.

Unas cuatro cosas que me gustaría que le agregaran de ser posible, no conozco la dificultad de hacerlo.

1. Lo primero sería que cuando se abra la app de FTP automáticamente se conecte al wifi, si es que ya está configurado, si no llega a estarlo que abra la app de wifi.

2. Lo otro es un indicador de volumen, como una barra lateral de 2px como lo tiene onion (sé que las comparaciones son odiosas pero esa implementación me resultó muy útil para la mm+, si se puede agregar otra para el brillo... 💯)

3. También una ventana para confirmar cuando se borra un juego, sé que la combinación de teclas para borrar no se pueden presionar de casualidad, pero a veces cuando estás borrando muchos juegos que no me gustan se me cuela alguno que no quería borrar... O tal vez la implementación de una _"papelera de reciclaje"_ que haya que vaciar después, pero que se pueda desactivar desde el menú de configuración (esto de la papelera se me ocurrió mientras escribía).

4. Y la cuarta que no sé qué tan difícil sería es tener dos perfiles de volumen, uno para cuando la consola está con auriculares y otro para cuando no. Generalmente juego en la noche con auriculares, pero cuando la enciendo sin ellos conectado me llevo una sorpresa jaja. En mi pc con linux funciona, desconozco los otros SO de pc, pero no sé si es por el controlador de sonido, el sistema, el kernel, etc. En fin, tampoco es tan urgente.

#### TL;RD

* App FTP conectar directamente al WIFI ya configurado.
* Indicador de volumen _(¿Y de brillo?)_.
* Confirmar al eliminar juegos _(¿Papelera de reciclaje?)_.
* Perfiles de volumen.

Una vez más, muchas gracias por el esfuerzo que hace todo el equipo, con los temas, la programación, las configuraciones de retroarch, los skrapeos, etc.

Abrazo.

**Comment by Ninoh-FOX** [comment]
Punto 1. Perderia todo el sentido la app wifi

Punto 2. Se va a implememtar en la 1.5

Punto 3. Simplemenu es un infierno de codigo

Punto 4. No se puede, ya que no hay un canal de audio de altavo y otro de auriculares, simplemente corta el altavoz a nivel hardware, ya que el sistema no manda ningun aviso al conectar auriculares.

**Comment by Ninoh-FOX** [comment]
Respuesta actualizada de la pregunta 4, el modelo flip si tiene audio digital, por lo que si tiene un canal de audio de altavoz y auriculares separados, por lo que se ha añadido la funcion en la version 1.6,1
